include/xten/Conversion/ATenToXTen.td

```diff
@@ -15,23 +15,23 @@ include "torch-mlir/Dialect/Torch/IR/TorchOps.td"
 
 include "xten/Dialect/XTen/XTenOps.td"
 
+// normalize names
```

Comment:
I like this. Can you add some more explanation as this can happen for more torch operations. (I think it comes from PyTorch in-place and by-return variants of the same operation having the same torch-mlir meaning but differing in name still due to the additional `_`).